### PR TITLE
bugfix: hide the loading indicator

### PR DESCRIPTION
**Pull request by dtngoc99**
## Change Description 👋

Hide the loading indicator

## Demo 📺

**[Loom Video Demo](https://loom.com/share/37964ae8cc5143d9ae1b809f7b7b7ce7)**

image

### CODE REVIEW

File: src/register/components/RegisterForm.vue [hunk 1]
```diff
@@ -70,7 +70,7 @@
 <script>
 import validators from "@/shared/form-validators";
 export default {
-  props: ["errorMessage", "successMessage"],
+  props: ["errorMessage", "successMessage", "isButtonDisabled"],
```
Comment (luantc13) [inline src/register/components/RegisterForm.vue:73]: We should declare the data type for props.

**Review by luantc13 — APPROVED** [review]
Great work!

File: src/register/components/RegisterForm.vue [hunk 2]
```diff
@@ -45,12 +45,11 @@
...
-      v-on:click="validateForm"
       type="submit"
-      :disabled="successMessage != ''"
+      :disabled="isButtonDisabled"
     >
-      <span v-if="!successMessage">SIGN UP</span>
-      <span v-if="successMessage"
+      <span v-if="!isButtonDisabled">SIGN UP</span>
+      <span v-else
         >Signing Up
```
Comment (truonghungit) [inline src/register/components/RegisterForm.vue:53]: `Signing Up` should be uppercase

**Review by truonghungit — APPROVED** [review]
Good job

**Pull request merged**